docker/manifest.tmpl

```diff
@@ -0,0 +1,31 @@
+image: plugins/drone-coverage-report:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
```

Comment:
remove `drone-` suffix and update all image references to `plugins/coverage-report`